extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.js

```diff
@@ -33,17 +110,118 @@ export class AmpStoryAudioSticker extends AMP.BaseElement {
         </div>
       </div>
     );
+
+    const pageEl = closestAncestorElementBySelector(
+      this.element,
+      'amp-story-page'
+    );
+    this.pageId_ = pageEl.getAttribute('id');
+
+    Services.storyStoreServiceForOrNull(this.win).then((storeService) => {
+      this.storeService_ = storeService;
+      this.storeService_.subscribe(StateProperty.MUTED_STATE, (muted) => {
+        this.onMutedStateChange_(muted);
+      });
+      this.initializeListeners_();
```

Comment:
You have to initialize the listeners in the `layoutCallback`

Reply:
I think the issue is waiting for the store service. There's a few ways to fix this, but if you return on layoutcallback the promise that you use to wait for the store service, it should work.